rocm-opencl-runtime/PKGBUILD

```diff
@@ -2,16 +2,17 @@
 
 _opencl_icd_loader_commit='978b4b3a29a3aebc86ce9315d5c5963e88722d03'
 
-pkgbase=rocm-opencl-runtime
-pkgname=(rocm-device-libs rocm-opencl-runtime)
+pkgname=rocm-opencl-runtime
 pkgver=3.3.0
 pkgrel=1
+pkgdesc='Radeon Open Compute - OpenCL runtime'
 arch=('x86_64')
 url='https://github.com/RadeonOpenCompute'
+license=('MIT')
+depends=("hsakmt-roct>=${pkgver}" "hsa-rocr>=${pkgver}" 'opencl-icd-loader')
```

Comment:
Is there a specific reason why need the pkgver? I would prefer to not have them.

Reply:
No, it was just copy and paste.